config/config.inc.php

```diff
@@ -69,6 +69,7 @@
 $config['scrollbar'] = false;
 $config['show_date'] = false; // only works if file_format_date = true
 $config['gallery']['date_format'] = 'd.m.Y - G:i';
+$config['slideshow_pictureTime'] = '3000';
```

Comment:
Why is this a string?

Reply:
Because of the `''` ? We've done that on all other config options too, added for consistency

Reply:
The picture quality is just a number, but you are right, most of the numerical values are strings. Usually this is no problem, just in js could be some misbehavior.